src/preferences/service.js

```diff
@@ -19,7 +19,7 @@ const Remote = imports.utils.remote;
 /*
  * Header for support logs
  */
-const LOG_HEADER = new GLib.Bytes(`
+const LOG_HEADER = new TextEncoder().encode(`
```

Comment:
```suggestion
const LOG_HEADER = new GLib.Bytes(`
```
This should stay `GLib.Bytes`, since it's passed to `Gio.OutputStream.write_bytes_async()`.